packages/matchbox/src/components/Tag/tests/Tag.test.js

```diff
@@ -1,28 +1,26 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import 'jest-styled-components';
 
 import Tag from '../Tag';
 
-describe('Tag', () => {
-  let wrapper;
-  beforeEach(() => {
-    const props = {
-      color: 'red'
-    };
+// Feels unecessary to test color variations here /shrug
 
-    wrapper = shallow(<Tag {...props}><span>Tag 1</span></Tag>);
+describe('Tag', () => {
+  it('should render a default tag', () => {
+    const wrapper = global.mountStyled(<Tag>Hola!</Tag>);
+    expect(wrapper).toHaveStyleRule('background', '#d9e0e6');
+    expect(wrapper).toHaveStyleRule('color', '#39444d');
   });
 
-  it('renders without close button', () => {
```

Comment:
Looks like we lost the test for when the remove button is *not* present? Probably worth preserving.

Reply:
Wasn't exactly sure what to assert on but I guess this works `expect(wrapper.find('button')).not.toExist();`

Reply:
That handles it for sure. Thanks.